setup.cfg

```diff
@@ -13,7 +13,7 @@ replace = __version__ = "{new_version}"
 
 [pycodestyle]
```

Comment:
Is this needed now that we have removed `pycodestyle`?

Reply:
Indeed it is not needed since we introduced Black. I'll delete the section.